entries/repo-website-overview.puml

```diff
@@ -0,0 +1,79 @@
+@startuml Repos-Sites
+
+  package .github as GIT {
+      diamond dia1
+  }
+
+  package data as DAT {
+      diamond dia2
+  }
+
+  package seedcase-theme as THE {
+      diamond dia3
+  }
+
+  package community as COM {
+      object "community.seedcase-project.org" as CSP {
+        Project blocks
+        Roadmap
+        Guides
+        Posts
+        Outreach
+        Code of Conduct
```

Comment:
I think at this point, it is best to not include all these here, since they will quickly get outdated and this will need to updated. Referencing the website should be enough right now.

Reply:
Done